src/transformers/modeling_roberta.py

```diff
@@ -758,13 +758,14 @@ def forward(
 
         Example::
 
-            >>> from transformers import RobertaTokenizer, RobertaLMHeadModel, RobertaConfig
+            >>> from transformers import RobertaTokenizer, RobertaForCausalLM, RobertaConfig
             >>> import torch
 
             >>> tokenizer = RobertaTokenizer.from_pretrained('roberta-base')
             >>> config = RobertaConfig.from_pretrained("roberta-base")
             >>> config.is_decoder = True
-            >>> model = RobertaLMHeadModel.from_pretrained('roberta-base', config=config, return_dict=True)
+            >>> config.return_dict = True
```

Comment:
We should be able to do
```
config = RobertaConfig.from_pretrained("roberta-base", return_dict=True)
```
a few lines up, no?

Reply:
Indeed!